Documentation/configuring-storage.md

```diff
@@ -50,6 +50,26 @@ data:
 ```
 The `spec.storage.hive.azure.container` should be the container you wish to store metering data at, and the optional `spec.storage.hive.azure.rootDirectory` should be the folder you want your data in, inside the container.
 
+## Storing data in Google Cloud Storage
+
+You can also store your data in Google Cloud Storage, and to do so, you must use an existing bucket.
+Edit the `spec.storage` section in the example [gcs-storage.yaml][gcs-config] configuration.
+Set the `spec.storage.hive.gcs.bucket` and `spec.storage.hive.gcs.secretName` with the secret following this format:
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: your-gcs-secret
+data:
+  gcs-service-account.json: "c2VjcmV0Cg=="
+```
+You can easily create the secret by calling:
+```
+kubectl -n $METERING_NAMESPACE create secret generic your-gcs-secret --from-file gcs-service-account.json=/path/to/your/service-account-key.json
+```
+Where $METERING_NAMESPACE is your namespace.
+The `spec.storage.hive.gcs.bucket` should be the bucket you wish to store metering data at, you can also append folder after the bucket name for the desired folder within the bucket. For example: metering-gcs/test
```

Comment:
The `spec.storage.hive.gcs.bucket` field should be the name of the bucket you wish to hold your metering data.

Reply:
You can also specify a sub-directory within this bucket by appending the name of the directory after the bucket name. For example: `metering-gcs/test`, where the bucket's name is `metering-gcs` and `test` is the name of the sub-directory.

Reply:
@chancez is this accurate/a better explanation?

Reply:
Yah that sounds good.